CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing to Telluric Deckay projects
+
+These contributing guidelines apply to all projects in the [Telluric
+Deckay](https://github.com/TelluricDeckay/) organization.
+
+To view open issues on all projects in the TD organization, you can use
+[this
+link](https://github.com/issues?q=is%3Aissue+is%3Aopen+org%3ATelluricDeckay).
+
+You may join our [Zulip](https://zulip.com/) chat room at
+[telluric-deckay.zulipchat.com](https://telluric-deckay.zulipchat.com/)
+
+## Bug Reports and Feature Requests
+
+Anyone may open an issue.
+
+## Coding Standards
+
+[Rust Code Style
+Guidelines](https://doc.rust-lang.org/book/appendix-04-useful-development-tools.html?highlight=formatting#automatic-formatting-with-rustfmt)
+
+## Patches and Pull Requests
+
+To prevent work-overlap, please post on a ticket if you'll be working
+on a specific issue (or create a ticket if there's not one open yet.
+**Note**: If more than one person submits a patch for the same thing,
+your patch may get rejected.
+
+**Note**: If you agreed to work to work on a ticket but later find that
+you're unable to work on it, or if you changed your mind, please post
+again on the ticket to let everyone know it's up for grabs.
```

Comment:
Might be best to state a time limit when the ticket reverts to unassigned again. E.g. If it's been over a week/ 10 days... or whatever since a progress update. (Unless a "reserve" time is agreed.)

Reply:
I just removed the section for now. All good now?

Reply:
Uh ok we'll go with that. Can change it later if needs be.